Homework_7.py

```diff
@@ -0,0 +1,203 @@
+
+
+class Buisnesman(Person):
+    def __init__(self):
+        Person.__init__(self)
+        self.company = str(input("Enter company: "))
+        self.meeting_at_arrival = str(input("Place of met: "))
+        self.special_requirements = str(input("Write special require: "))
+        self.special_offers_info = {'company': self.company,
+                                    'met': self.meeting_at_arrival,
+                                    'sp_req': self.special_requirements}
+        Person.change_room(self)
+
+    def get_spec_info(self):
+        return self.special_offers_info
+
+    def base_info(self):
+        self.person_info = {'name': self.name,
+                            'age': self.age,
+                            'sex': self.sex,
+                            'name_room': self.name_room,
+                            'num_of_night': self.num_of_night}
+        return self.person_info
+
+
+class Sportsman(Person):
+    def __init__(self):
+        Person.__init__(self)
+        self.rent_sport_places = str(input("Place of sport_place: "))
+        self.inventory = str(input("Enter inventory: "))
+
+        self.spec_offers_info = {'rent': self.rent_sport_places,
+                                 'inventory': self.inventory}
+        Person.change_room(self)
+
+    def base_info(self):
+        self.person_info = {'name': self.name,
+                            'age': self.age,
+                            'sex': self.sex,
+                            'name_room': self.name_room,
+                            'num_of_night': self.num_of_night}
+        return self.person_info
+
+    def get_spec_info(self):
+        return self.spec_offers_info
+
+
+class Hotel(object):
+    def __init__(self):
+        self.name = str(input("Enter name of hotel: "))
+        self.rang = int(input("Enter stars of hotel: "))
+        self.base_info = {'Name': self.name, 'Rang': self.rang}
+
+    prices_room = {"room_1": 100,
+                   "room_2": 110,
+                   "room_3": 150,
+                   "room_4": 200,
+                   "room_5": 115,
+                   "room_6": 110,
+                   "room_7": 150,
+                   "room_8": 140
+                   }
+
+    def info_room(self):
+        return self.prices_room
+
+    def select_room(self, prices_room):
+        room = str(input("Select the room: "))
+        select_room = prices_room.get(room, "Unknown_room")
+        print("Select {}, price: {}".format(room, select_room))
+
+    def add_new_room(self, prices_room):
+        name = str(input("Enter name of room: "))
+        base_price = int(input("Enter price: "))
+        new_room = {name: base_price}
+        prices_room.update(new_room)
+        print("New room: {}, price: {}".format(name, base_price))
+        return prices_room
+
+    # change base price of room
+    def update_base_price(self):
+        room = str(input("Enter name of room: "))
+        price = int(input("Enter new price: "))
+
```

Comment:
can't we remove this and add_new_room method and make only one with 
self.prices_room[room] = price?